tests/test_idpi/test_potvortic.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import xarray as xr
+import pytest
+
+from idpi import grib_decoder
+import idpi.operators.pot_vortic as pv
+
+
+@pytest.fixture
+def fieldextra_executable(fieldextra_executable):
```

Comment:
Here is an example of the fixture being overridden at the module level.

[] remove this before merging